.classpath

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file (along with a few other) should not be in git, but should be ignored using a `.gitignore` file. If you take the file from the pulse-us/api project and use that it'd be a great starting point: https://github.com/pulse-us/api/blob/development/.gitignore